mlflow/protos/model_registry.proto

```diff
@@ -230,9 +230,13 @@ service ModelRegistryService {
   rpc setRegisteredModelTag (SetRegisteredModelTag) returns (SetRegisteredModelTag.Response) {
     option (rpc) = {
       endpoints: [{
+        method: "POST",
+        path: "/mlflow/registered-models/set-tag"
```

Comment:
We will maintain both preview and non-preview paths for one release to ensure that clients which are one version out-of-date are still compatible with servers running the latest version.